src/Microsoft.Azure.WebJobs.Host/JobHost.cs

```diff
@@ -85,6 +85,14 @@ public JobHost(JobHostConfiguration configuration)
             _stoppingTokenSource = CancellationTokenSource.CreateLinkedTokenSource(_shutdownTokenSource.Token);
         }
 
+        /// <summary>
+        /// Gets the JobHostConfiguration of this JobHost
+        /// </summary>
+        public JobHostConfiguration Configuration
```

Comment:
Yes we can, since Typelocator will no longer be needed to find MethodInfos for filter functions. The new IJobMethodInvoker interface will take a string name. Internally it will do the string => MethodInfo conversion.